src/applications/yellow-ribbon/containers/YellowRibbonApp/index.jsx

```diff
@@ -103,13 +98,50 @@ export const YellowRibbonApp = ({ results }) => (
           {results && (
             <h3 className="vads-u-margin-top--1p5">Search criteria</h3>
           )}
+
           <SearchForm />
+
+          {results && (
+            <>
+              <h3>Search criteria</h3>
+              <p>
+                <a href="/education/eligibility/" rel="noopener noreferrer">
+                  Find out if you&apos;re eligible for the Post-9/11 GI Bill
+                </a>
+              </p>
+              <p>
+                <a
+                  href="/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program/"
+                  rel="noopener noreferrer"
+                >
+                  Find out if you qualify for the Yellow Ribbon Program
+                </a>
+              </p>
+              <p>
+                <a href="/education/how-to-apply/" rel="noopener noreferrer">
+                  Apply for Post-9/11 GI Bill benefits
+                </a>
+              </p>
+            </>
+          )}
         </div>
 
         {/* Search Results */}
         <div className="vads-l-col--9 vads-u-padding-left--4">
           <SearchResults />
         </div>
+
+        {/* Post-Form Content */}
+        {!results && (
+          <p className="vads-l-col--7">
+            Participating school information is for the current academic year.
+            To view schools for the previous academic year,{' '}
+            <a href="" rel="noreferrer noopener">
+              download the historical rates (PDF)
+            </a>
+            .
+          </p>
```

Comment:
Is it definitely necessary for `results` to be populated before we can show the links? Seems odd to hide that stuff before a search.

Reply:
Great question actually, it shouldn't really be dependent on whether we have `results` but more on whether we've made a request yet for YRPs... 🤔 hmm

Reply:
For example, for a search with no results, all of a sudden this will not display which is not intended

Reply:
Updated!